bin/push/push_remind.py

```diff
@@ -1,13 +1,59 @@
+import arrow
 import json
 import logging
 import os
 import requests
+import sys
 
+import emission.core.get_database as edb
+import emission.storage.decorations.analysis_timeseries_queries as esda
 import emission.storage.decorations.user_queries as esdu
+import emission.storage.timeseries.timequery as estt
 import emission.net.ext_service.push.notify_usage as pnu
 
 STUDY_CONFIG = os.getenv('STUDY_CONFIG', "stage-program")
 
+
+def users_without_recent_user_input(uuid_list, recent_user_input_threshold=None):
```

Comment:
future fix: it should be possible to write unit tests for both these function, particularly if we want to reuse them later.